cli/stream_command.go

```diff
@@ -704,9 +706,18 @@ func (c *streamCmd) viewAction(_ *fisk.ParseContext) error {
 			if len(msg.Data) == 0 {
 				fmt.Println("nil body")
 			} else {
-				fmt.Println(string(msg.Data))
-				if !strings.HasSuffix(string(msg.Data), "\n") {
-					fmt.Println()
+				data := msg.Data
+				var errFilter error
```

Comment:
no real value in making new error variables here, just use the one we have. The logic here seems duplicated 3 times I think we can probably extract it into a generic helper function? Also worth imprving on the errors that wold jsut be `Error: ...` a better message would be clearer

Reply:
We don't allow variable shadowing in our codebase. Having names for errors also makes the code more readable, so this is out of habit. I can change that of course.